<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgCompositionDocker</class>
 <widget class="QWidget" name="WdgCompositionDocker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QListView" name="compositionView">
     <property name="toolTip">
      <string>Checked compositions will be exported, unchecked won't.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="saveNameEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="exportCompositions">
     <property name="toolTip">
      <string>Export checked compositions.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
